appinventor/components/src/com/google/appinventor/components/runtime/util/FroyoUtil.java

```diff
@@ -153,6 +153,15 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
       public void onPageFinished(WebView view, String url) {
         EventDispatcher.dispatchEvent(component, "PageLoaded", url);
       }
+
+      @Override
+      public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
+        container.$form().runOnUiThread(new Runnable() {
```

Comment:
In this context, `form` is available as a final variable.

Reply:
Looks like I forgot to add my changes to git. Whoops!